$NetBSD: patch-configure,v 1.6 2020/01/22 18:53:13 adam Exp $

Remove -Werror from default flags since different compilers
have very different warnings.

--- configure.orig	2020-01-21 22:09:10.000000000 +0000
+++ configure
@@ -7245,7 +7245,7 @@ $as_echo "#define _GNU_SOURCE 1" >>confd
 elif test "$GCC" = "yes"
 then
   GCC_VERSION=`$CC -dumpversion`
-  CFLAGS="$CFLAGS -Wall -Werror -pedantic -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls"
+  CFLAGS="$CFLAGS -Wall -pedantic -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls"
   if test "x$enable_asan" = "xyes"; then
     CFLAGS="$CFLAGS -fsanitize=address"
   fi
